tiledb/ml/readers/_tensor_schema.py

```diff
@@ -106,7 +75,11 @@ def shape(self) -> Sequence[int]:
     @property
     def query(self) -> KeyDimQuery:
         """A sliceable object for querying the TileDB array along the key dimension"""
-        return KeyDimQuery(self._array, self._key_dim_index, **self._query_kwargs)
+        return KeyDimQuery(
+            self._array,
+            self._key_dim_index,
+            **self._query_kwargs,
+        )
```

Comment:
[formatting] this can be one line as it was after removing the trailing comma after `self._query_kwargs`